config/7.1.0/securebanking/ig/scripts/groovy/RepoUser.groovy

```diff
@@ -14,39 +14,60 @@ userRequest.setMethod('GET');
 response = new Response(Status.OK)
 response.headers['Content-Type'] = "application/json"
 
-def splitUri =  request.uri.path.split("/")
-
-if (splitUri.length == 0) {
-    message = "Can't parse api client ID from inbound request"
+// split uri by filter request param
+def uriFilter = request.uri.toString().split("\\?")
+// split uri path to validate it
+def splitUriPath =  request.uri.path.split("/")
```

Comment:
You can use: `org.forgerock.http.MutableUri#getPathElements`

Reply:
Good suggestion, done! I have added some improvements also to validate properly the values from the request URI